### ISSUE TITLE
spring boot 2.0.1 以上有这样的问题，使用小心啦，会翻译的给他们翻译

### ISSUE THREAD

**Issue by lanmingle** [spring-projects/spring-boot#13633]
spring boot 2.0.1.RELEASE 以上（测试到2.0.3）会出现 nginx 转发出现 400 ，错误原因未知，spring debug 没有响应过日志，但是通过 tcpdump 能收到到  nginx 转发过来的请求。

后来把版本降到 2.0.1.RELEASE 一切正常。

**Comment by wilkinsona** [comment]
Courtesy of Google translate:

```
Spring boot 2.0.1.RELEASE The above (test to 2.0.3) will appear nginx forwarding occurs 400, the cause of the error is unknown, spring debug did not respond to the log, but through tcpdump can receive nginx forwarded over the request.

Later version down to 2.0.1.RELEASE everything is normal.
```

Unfortunately, there's no way to tell for certain what the problem may be from the description you have provided. However, I would guess that this is a duplicate of https://github.com/spring-projects/spring-boot/issues/13236 which is actually a Tomcat problem. The fix in Tomcat is in 8.5.32 which we'll pick up before the next Boot 1.5.x and 2.0.x releases.

**Comment by lanmingle** [comment]
@wilkinsona Thank you. The problem has been confirmed. It is indeed the problem.
